### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by synssins** [cp2004/OctoPrint-WLED#23]
Please add the ability to select multiple WLED instances, and for each effect, which instance you want it to display on. 

I have numerous WLED lights throughout the house. I'd like to be able to set a print status/progress alert on multiple lights in order to see the status from anywhere.

**Comment by cp2004** [comment]
Maybe in the longer term future - this adds an additional dimension to the plugin, from multiple segments to multiple segments *on each device*. The plugin is already *fairly* complex to configure to begin with, it will a fair bit of work to add in a way that doesn't make it complex.

**Comment by droptopgokart** [comment]
I would be fine with just being able to assign multiple WLED instances, to Octoprint.  The different effects for each WLED instance wouldn't be an issue for most, I wouldn't think.

**Comment by cp2004** [comment]
Agreed that could be an interim solution, to connect to multiple WLED devices, but have the same effects on each. With this plugin there's a lot more care that has to go in, because it is an external device over the network. I will think about it.

**Comment by MrTechGadget** [comment]
This feature would be cool, but personally I use Home Assistant and Node Red with Octoprint and WLED integrations, so for my secondary (or more) WLED devices I could use automations to set those and use this for the strip at my printer.

**Comment by nibblerrick** [comment]
Nice that I am not the only one thinking of this.
Multiple WLED-Strips with different effect-assignments is truely another level...
But to be able "just" to sent out the config to more then one instance would be very handy.
E.g. my printer is in another floor and having one strip near the printer and one on another floor to have the status shown would be very welcome...at least for me :-D

**Comment by niklas2244** [comment]
you can youse the SYNC feature of WLED in the App to Sync them up. No need to adjust the plugin

**Comment by Fabaa96** [comment]
I have the same request, sending via the plugin to multiple devices. 
If I SYNC the connected lamp to others all of my lamps in every room make the same and thats a little bit too much.
Thanks for the awesome plugin. :)
